### PR TITLE
site: add examples for v1alpha3

### PR DESCRIPTION
**Pull request by neolit123**
- add minimal multi CP example
- update the example YAML to v1alpha3
- add API redirect

fixes https://github.com/kubernetes-sigs/kind/issues/266

/kind documentation
/priority important-longterm

### CODE REVIEW

File: site/content/docs/user/kind-example-config.yaml [hunk 1]
```diff
@@ -1,30 +1,29 @@
...
+kind: Cluster
+apiVersion: kind.sigs.k8s.io/v1alpha3
+# patch the generated kubeadm config with some extra settings
+kubeadmConfigPatches:
+- |
+  apiVersion: kubeadm.k8s.io/v1beta1
+  kind: ClusterConfiguration
+  metadata:
+    name: config
+  networking:
```
Comment (aojea) [inline site/content/docs/user/kind-example-config.yaml]: v1alpha3 is going to have a new `networking` field #340 .

I've sent a proposal (#348)  using ` serviceSubnet` and `podSubnet` in the new `networking` section.

Does it make sense?

Comment (neolit123) [reply site/content/docs/user/kind-example-config.yaml]: depends on what we want in the kind example config file(s).

File: site/content/docs/user/quick-start.md [hunk 2]
```diff
@@ -202,12 +202,26 @@ In particular, many users may be interested in multi-node clusters. A simple
 configuration for this can be achieved with the following config file contents:
 ```yaml
 # three node (two workers) cluster config
```
Comment (BenTheElder) [inline site/content/docs/user/quick-start.md]: this comment doesn't match now

Comment (neolit123) [reply site/content/docs/user/quick-start.md]: hm, it says 3 nodes total, with 2 workers.
```
# three node (two workers) cluster config
kind: Cluster
apiVersion: kind.sigs.k8s.io/v1alpha3
nodes:
- role: control-plane
- role: worker
- role: worker
```

should this be changed?

Comment (BenTheElder) [reply site/content/docs/user/quick-start.md]: Derp, thanks

File: site/content/docs/user/kind-example-config.yaml (hunk 1 above)
Comment (BenTheElder) [reply site/content/docs/user/kind-example-config.yaml]: Let's only add what is merged for now with the API server options and come back to discussing these other options (next meeting / 0.3 planning perhaps?)

Comment (neolit123) [reply site/content/docs/user/kind-example-config.yaml]: ok, i'd prefer if we talk about this PR the next meeting and see if this config is what we want for the next release.

Comment (BenTheElder) [reply site/content/docs/user/kind-example-config.yaml]: SGTM, 0.2 is about ready to go, would be good to document v1alpha3 for that and add in these extra fields later.

Comment (BenTheElder) [reply site/content/docs/user/kind-example-config.yaml]: .2 mostly needs relnotes and https://github.com/kubernetes-sigs/kind/pull/394 https://github.com/kubernetes-sigs/kind/pull/393

Comment (neolit123) [reply site/content/docs/user/kind-example-config.yaml]: so is this PR good for 0.2?

**Comment by BenTheElder** [comment]
let's add our networking field in a follow-up

**Pull request merged**